src/components/cookie-banner/index.md.njk

```diff
@@ -106,8 +106,8 @@ Here's the same example of a progressively enhanced cookie banner, with the conf
 ### Option 3: If you set non-essential cookies, but only on the client
 
 You can choose to make your banner only work with JavaScript if your service only needs to set non-essential cookies on the client.
- 
-When the page loads, include all the cookie banner messages that the user might see, but hide these with the `hidden` HTML attribute.
+
+When the page loads, hide both the component (`govuk-cookie-banner`) and all the cookie banner messages (`govuk-cookie-banner__message`) the user might see, using the `hidden` HTML attribute.
```

Comment:
I'm not sure about this... I find it a little confusing.

I think in part it might be that having the verb 'hide' implies that an action is taken, where in fact it is a lack of action that keeps the cookie banner hidden.

Reply:
Also, I'm not sure how helpful the class names are; while both these classes do need to be hidden, my understanding is that `govuk-cookie-banner__message` should always be inside a `govuk-cookie-banner`, so you shouldn't need the `hidden` attribute on both?

I could be wrong about that though.

Reply:
Thanks for reviewing!

I think Hanna's concern was that users might not understand that, by 'component', we mean `govuk-cookie-banner`, and by 'message', we mean `govuk-cookie-banner__message`. But I'm not sure whether that's something a user reported or just a hunch about something that _could_ happen.

Would rewording like this avoid implying the user takes an action?

> When the page loads, the `hidden` html attribute hides the component, as well as all the cookie banner messages it contains, which the user might otherwise see.

Reply:
@calvin-lau-sig7 Can I get a review? :)

Reply:
The first re-wording reads better to me!

Reply:
Cheers, @lfdebrux!